src/Platforms/SQLitePlatform.php

```diff
@@ -561,6 +562,16 @@ public function getCreateIndexSQL(Index $index, string $table): string
             ));
         }
 
+        if ($index->isFunctional() && ! $this->supportsFunctionalIndex()) {
```

Comment:
We're inside a platform class for SQLite. We should _know_ whether SQLite supports functional indexes.